test/integration/IntegrationTestListener.php

```diff
@@ -42,12 +41,17 @@ public function startTestSuite(TestSuite $suite)
             $this->fixtureLoader = new \ZendIntegrationTest\Db\Platform\PgsqlFixtureLoader();
         }
 
+        if (! isset($this->fixtureLoader)) {
+            return;
+        }
+        printf("\nIntegration test started.\n");
         $this->fixtureLoader->createDatabase();
     }
 
     public function endTestSuite(TestSuite $suite)
     {
-        if ($suite->getName() !== 'integration test') {
+        if ($suite->getName() !== 'integration test' ||
+            ! isset($this->fixtureLoader)) {
```

Comment:
This should read:

```php
if ($suite->getName() !== 'integration test'
    || ! isset($this->fixtureLoader)
) {
```

(Operators go on the next line, so that we can add more conditions without requiring additional lines of changes.)

I'll fix this on merge.